avail/src/main/kotlin/avail/compiler/splitter/MessageSplitter.kt

```diff
@@ -1262,7 +1484,28 @@ constructor(messageName: A_String) {
 
 	override fun toString() = buildString { dumpForDebug(this) }
 
-	companion object {
+	companion object
+	{
+		/**
+		 * The public factory for splitting message names.  The result may be
+		 * cached in an [LRUCache].  Since the [messageName] is used as a key in
```

Comment:
You reference the value parameter, `messageName`, in a document link but you do not provide an `@param` comment description for `messageName`. When clicked through in the IDE, it provides no information other than type information.